src/sorts.tsx

```diff
@@ -64,6 +64,18 @@ const SORTS: Sort[] = [
     name: "Callsign",
     impl: (a, b) => a.callsign.localeCompare(b.callsign),
   },
+  {
+    name: "Type",
+    impl: (a, b) => a.cardType.localeCompare(b.cardType),
+  },
+  {
+    name: "Location",
+    impl: (a, b) => a.location && b.location ? a.location.localeCompare(b.location) : a.location ? -1 : b.location ? 1 : 0
+  },
+  {
+    name: "Frequency",
```

Comment:
good opportunity to add tags to divide the column view into bands (2m, 1.25m, 70cm...)